### ISSUE TITLE
Pre-downtime indicator

### ISSUE THREAD

**Issue by kadrika** [centreon/centreon#4483]
Hi,

Can you please add an enhancement that if service or host has been configured to be in downtime in the future (so that the downtime has not yet started) then there will be different icon showing that this host or service will be in downtime state during next  5 day or a week. And if possible make it so that this enhancement can be enabled/disabled globally in frontend configuration, because I am quite sure there are people who do not want to see this. 

It is quite annoying to go through the downtime list to check if all the needed hosts/services have the downtime for example for the next maintenance window.

**Comment by garnier-quentin** [comment]
Good idea. We put it in details page (in a tab). We can't display it in tabular (it will cost too much in performance).

**Comment by kadrika** [comment]
Can it be so that the exclamation mark icon will be green if there is downtime added for the future and when the downtime starts it will be changed to that lilac/pink (or what ever color that)  icon.
